lib/twilio-ruby/twiml/voice_response.rb

```diff
@@ -0,0 +1,494 @@
+        wait_url: nil,
+        wait_url_method: nil,
+        workflow_sid: nil,
+        **keyword_args)
+
+        self.append(Enqueue.new(
+          name,
+          action: action,
+          method: method,
+          wait_url: wait_url,
+          wait_url_method: wait_url_method,
+          workflow_sid: workflow_sid,
+          **keyword_args
+        ))
+      end
+
+      def gather(
+        action: nil,
+        method: nil,
+        timeout: nil,
+        finish_on_key: nil,
+        num_digits: nil,
+        partial_result_callback: nil,
+        partial_result_callback_method: nil,
+        language: nil,
+        hints: nil,
+        barge_in: nil,
+        acknowledge_sound_url: nil,
+        **keyword_args)
+
+        self.append(Gather.new(
+          action: action,
+          method: method,
+          timeout: timeout,
+          finish_on_key: finish_on_key,
+          num_digits: num_digits,
+          partial_result_callback: partial_result_callback,
+          partial_result_callback_method: partial_result_callback_method,
+          language: language,
+          hints: hints,
+          barge_in: barge_in,
+          acknowledge_sound_url: acknowledge_sound_url,
+          **keyword_args
+        ))
+      end
+
+      def hangup()
+        self.append(Hangup.new)
+      end
+
+      def leave()
+        self.append(Leave.new)
+      end
+
+      def pause(length: nil)
+        self.append(Pause.new(length: length))
+      end
+
+      def play(url, loop: nil, digits: nil, **keyword_args)
+        self.append(Play.new(
+          url,
+          loop: loop,
+          digits: digits,
+          **keyword_args
+        ))
+      end
+
+      def record(
+        action: nil,
+        method: nil,
+        timeout: nil,
+        finish_on_key: nil,
+        max_length: nil,
+        play_beep: nil,
+        trim: nil,
+        recording_status_callback: nil,
+        recording_status_callback_method: nil,
+        transcribe: nil,
+        transcribe_callback: nil,
+        **keyword_args)
+
+        self.append(Record.new(
+          action: action,
+          method: method,
+          timeout: timeout,
+          finish_on_key: finish_on_key,
+          max_length: max_length,
+          play_beep: play_beep,
+          trim: trim,
+          recording_status_callback: recording_status_callback,
+          recording_status_callback_method: recording_status_callback_method,
+          transcribe: transcribe,
+          transcribe_callback: transcribe_callback,
+          **keyword_args
+        ))
+      end
+
+      def redirect(url, method: nil, **keyword_args)
+        self.append(Redirect.new(url, method: method, **keyword_args))
+      end
+
+      def reject(reason: nil, **keyword_args)
+        self.append(Reject.new(reason: reason, **keyword_args))
+      end
+
+      def say(body, loop: nil, language: nil, voice: nil, **keyword_args)
+        self.append(Say.new(
+          body,
+          loop: loop,
```

Comment:
I don't think so, but yeah the syntax highlighting is weird.

https://docs.ruby-lang.org/en/2.4.0/keywords_rdoc.html